src/BookReader/Mode1UpLit.js

```diff
@@ -1,5 +1,5 @@
 // @ts-check
-import { customElement, html, LitElement, property, query } from 'lit-element';
+import { customElement, html, LitElement, property, query } from 'lit';
```

Comment:
`customElement` and `property` are in a different place; see https://lit.dev/docs/releases/upgrade/#update-decorator-imports

Reply:
☑  `customElement` and `property` are 
       set to be imported from 
      `lit/decorators.js`